command_line/sweep_to_stills.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python
+#
+# LIBTBX_SET_DISPATCHER_NAME dials.sweep_to_stills
+
+from __future__ import absolute_import, division, print_function
+
+import logging
+
+from dxtbx.model.experiment_list import Experiment, ExperimentList
+from scitbx.matrix import sqr, col
+from dxtbx.model import MosaicCrystalSauter2014
+from cctbx import crystal_orientation
+from dials.model.data import Shoebox
+from dials.algorithms.refinement.prediction.managed_predictors import (
+    ExperimentsPredictorFactory,
+)
+from dials.util import show_mail_on_error
+from libtbx.phil import parse
+
+logger = logging.getLogger("dials.command_line.stills_process")
+
+help_message = """
+
+Utility script to split a sweep into a set of stills.
```

Comment:
```suggestion
Split a sweep into a set of stills.
```
The first 3 words add no useful information to the program description.